src/compute_shader.rs

```diff
@@ -176,6 +177,7 @@ impl<S: ComputeShader> FromWorld for ComputeShaderPipeline<S> {
             shader_defs: vec![],
             entry_point,
             push_constant_ranges: vec![],
+            zero_initialize_workgroup_memory: true,
```

Comment:
This was the recommended default. I don't have any specific knowledge about the different behaviour.